magefile.go

```diff
@@ -32,7 +32,7 @@ var (
 
 	publicDir    = "./public"
 	buildDir     = "./build"
-	packagePaths = []string{"./dev/packages/alpha/", "./dev/packages/example/"}
+	packagePaths = []string{"./dev/packages/storage/", "./dev/packages/example/"}
```

Comment:
We could add this directory to gitignore and then have the magefile to also get the storage dependencies in. The nice part would be that then also `go run .` works with all of them.

Reply:
I will adjust the PR

Reply:
Done